crates/service-protocol/Cargo.toml

```diff
@@ -12,21 +12,21 @@ default = []
 codec = ["dep:restate-types", "dep:paste"]
 discovery = ["dep:serde", "dep:serde_json", "dep:bytestring", "dep:regress", "dep:tracing", "dep:codederror", "dep:restate-errors", "dep:http", "dep:http-body-util", "dep:restate-service-client", "dep:restate-types", "dep:tokio"]
 message = ["dep:restate-types", "dep:bytes-utils", "dep:codederror", "dep:restate-errors", "dep:size", "dep:tracing"]
-test-util = []
+test-util = ["codec", "restate-types/test-util"]
```

Comment:
Why is "codec" needed here?

Reply:
Thank you for pointing that out. It was unnecessary, and I have already deleted it.